src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int RANDOM_NUMBER = 101;
+    private ColorSupplier colorSupplier;
+
+    public Lottery() {
+        this.colorSupplier = new ColorSupplier();
+    }
+
+    public ColorSupplier getColorSupplier() {
+        return colorSupplier;
+    }
+
+    public void setColorSupplier(ColorSupplier colorSupplier) {
+        this.colorSupplier = colorSupplier;
+    }
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+
+        String randomColorName = colorSupplier.getRandomColor().name();
+        Color randomColor = Color.valueOf(randomColorName);
```

Comment:
why do you wrap and then unwrap color value?
```suggestion
        Color randomColor = colorSupplier.getRandomColor();
```